### DISCUSSION TITLE
I am very confused about hardware and how to set up a project...can someone update the ultimate hyperhdr guide?

### DISCUSSION THREAD

**Discussion by X-15** [awawa-dev/HyperHDR#850] [Q&A]
Hello everyone,

I've been reading the 2023 Ultimate Guide multiple times, but I'm still confused about how to proceed. It seems like many things have changed, including the recommendations, leaving me feeling quite lost. I really need a straightforward guide on what to buy and how to solder (if needed). While the Ultimate Guide explains how everything works or doesn't, I can't find a practical, hands-on guide to help me build the hardware from scratch.

Currently, my Windows PC is my video source, and I've installed HyperHDR. I've successfully synced it to my Philips Hue Bridge, which allows the lights to follow my TV output. This setup works, but now I'd like to add an LED strip to my 85" TV, connecting the strip to a controller and Raspberry Pi. Initially, I want to try running it via Wi-Fi to see its performance. If it's not up to par, I can use a USB grabber-HDMI splitter, whichever is currently recommended which I also don't know if I want to keep 4k and all the fancy hdmi features.

Unfortunately, I haven't found a comprehensive, easy-to-follow guide for this. Could anyone kindly provide direct instructions? I don't want to rely on guesswork or assume a product will work because it seems logical on paper. I also don't have the patience or expertise to guess about soldering correctly.

I would sincerely appreciate any assistance you can provide.

Thank you so much!

**Comment by awawa-dev** [comment]
Hi
The only major thing that has come out since the last guide and has changed things a bit (simplified setup since some rp2040s have a built-in level shifter so it's ideal if you dont have a patience) is HyperSerialPico but the guide mention about it. More details can be found on HyperSerialPico project and it's very similar to HyperSerialESP32 only rp2040 is better and easier to flash than esp32. The rest is valid and very detailed described. The most frequently chosen configurations when it comes to the grabber are ezcoo splitter + ms2109 (if we only have USB2.0) or ms2130 (usb3.0). Or Elgato HD60x as the all in one solution (if we do not need HDCP and Dolby Vision LLDV support). There are many similar topics here.

**Reply by X-15** [reply]
Ok....let's say I buy this one https://www.adafruit.com/product/5650 ....sorry for asking dumb, noob questions, but thats how I feel right now...what then? Yes I flash the pico firmware on it, but how do I connect the LED Strip physically to it? All I see are some holes on the board on the product page...what kind of connecters do I need?  How do I power the device and the cables? etc. These are detailed questions which decide whether the project can be done or not...Somebody like me needs a detailed hands on guide to do it properly and I think lack of such a guide is what scaring most people getting into this diy board stuff. Pico legit seems like an easier alternative especially without the shifters and yet still so many unanswered questions and I can't find any answers elsewhere....So what is my alternative? Buying fancy leds even if they fail to sell rgbw strips? hahah 

I am entitled to such a guide? No. 
Am I demanding it? No. 
Am I lost without such a guide? Yes.
Would it help tons of people like me? Yes.

**Reply by awawa-dev** [reply]
I've already created 2 huge and detailed tutorials over the last few years, and the last one is still relevant. So I think that's enough for me and more depends on the community: you're welcome. For the Pico Adafruit board, connect the signal line and ground from Pico to the supported LED strip. That's all for connecting this LED driver.

**Reply by awawa-dev** [reply]
![obraz](https://github.com/awawa-dev/HyperHDR/assets/69086569/b8662eed-5077-47bf-bc7c-c4b142d81237)
Still you need to read manual for HyperSerialPico

**Reply by X-15** [reply]
I figured the gpios out but the "how" is what I am missing, I never worked with such boards...I have no idea what connecters are suited for LED Strips...do LED Strips come with needed connecters for the driver? Do I strip the LED strip connector down to the stripped wire and put them inside the hole on the board? Are there adapters I can buy? Those questions might seem stupid to you, but how would I know it if nobody explains it. In ultimate guide those questions arent answered either. I've read the long manual on adafruits page, but I still don't know what the correct way to connect the LED strips would be, because their explanations are very broad and are aimed to a crowd who know what they are doing... and I am sorry for not having too much experience with this, but it is what it is.

**Reply by adworacz** [reply]
@X-15 you might be served well be watching an example video like this one https://www.youtube.com/watch?v=bhWJIAqG7p0

The setup may not be *exactly* the same, but it should answer a lot of your questions on things like wiring, connecting LED strips, etc.
